statsmodels/iolib/table.py

```diff
@@ -878,7 +878,8 @@ def set_datatype(self, val):
     stub=r'\textbf{%s}',
     empty='',
     missing='--',
-    replacements={"%" : "\%", ">" : "$>$", "|" : "$|$"}
+    #replacements will be processed in lexicographical order
```

Comment:
$ has to be replaced first I would expect, else you end up replacing inserted $'s.